ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md

```diff
@@ -1156,6 +1199,96 @@
                 #   - 1-10
               - range: <str; required; unique>
                 mac_address_maximum: <int>
+          tap:
+
+            # VLAN ID or range of VLAN IDs within range 1-4094.
+            allowed_vlan: <str>
+            default:
+
+              # Tap group names for the interface.
+              groups:
+                - <str>
+
+              # Interfaces like -  Ethernet, InternalRecirc, Port-Channel, Recirc-Channel.
+              interfaces:
+                - <str>
+
+              # Default nexthop-group names.
+              nexthop_groups:
+                - <str>
+            identity:
+
+              # Tap port ID tag.
+              port_id: <int; 1-4094>
+
+              # Inner tap port ID tag.
+              inner_port_id: <int; 1-4094>
+
+            # Pop all MPLS labels.
+            mpls_pop_all: <bool>
+
+            # Native VLAN ID when interface is in tap mode.
+            native_vlan: <int; 1-4094>
+            truncation:
+              enabled: <bool; required>
+
+              # Ingress packet truncation size in bytes.
+              size: <int; 100-9236>
+            mac_address:
+
+              # MAC address for the source.
+              src: <str>
+
+              # MAC address for the destination.
+              dest: <str; required>
+            encapsulation:
+
+              # Strip VXLAN encapsulation header.
+              vxlan_strip: <bool>
+
+              # Strip GRE encapsulation header.
+              gre_strip: <bool; required>
```

Comment:
Lets discuss this option on call, I see more options here

Reply:
tap header option

Reply:
I am not able to see it in the running-config, thats not added it to schema

Reply:
```
s1-leaf1(config-if-Et2)#switchport tap encapsulation gre destination 1.1.1.1 source 2.2.2.2 protocol 0x0000 feature header length 3 strip 
s1-leaf1(config-if-Et2)#show active
interface Ethernet2
   description P2P_LINK_TO_S1-SPINE1_Ethernet2
   mtu 1500
   no switchport
   ip address 172.30.255.1/31
   switchport tap encapsulation gre destination 1.1.1.1 source 2.2.2.2 protocol 0x0 strip
```